database/mongo.js

```diff
@@ -41,19 +41,19 @@ var save = (review) => {
   return new Promise((resolve, reject) => {
     Review.create(review, (err, result) => {
       if (err) {
-        reject(err);
+        reject(err.stack);
       } else {
         resolve(result);
       }
     })
   })
 }
 
-var getHotelReviews = (hotelId) => {
+var read = (hotelId) => {
   return new Promise((resolve, reject) => {
     Review.find({"responderInfo.hotelId": hotelId}).exec((err, results) => {
       if (err) {
-        reject(err);
+        reject(err.stack);

```

Comment:
I've never seen this before! Does it just return the full stack trace when the error is thrown?

Reply:
Yes, it gives a more detailed error by showing the stack trace.